custom_components/mojelektro/const.py

```diff
@@ -4,6 +4,6 @@
12.0.0.0.0.0.0.0.0.3.72.0", "readingTypeObracun": "8.0.4.1.19.2.12.0.0.0.0.0.0.0.0.3.72.0", "tip": "delovna oddaja ET", "vrsta": "STANJE" }, { "naziv": "Oddana delovna energija VT", "opis": "24 urno stanje, A-, kWh, T1", "oznaka": "A-_T1", "perioda": "24 h", "readingType": "32.0.4.1.19.2.12.0.0.0.0.1.0.0.0.3.72.0", "readingTypeBrezObracuna": "0.0.4.1.19.2.12.0.0.0.0.1.0.0.0.3.72.0", "readingTypeObracun": "8.0.4.1.19.2.12.0.0.0.0.1.0.0.0.3.72.0", "tip": "delovna oddaja VT", "vrsta": "STANJE" }, { "naziv": "Oddana delovna energija MT", "opis": "24 urno stanje, A-, kWh, T2", "oznaka": "A-_T2", "perioda": "24 h", "readingType": "32.0.4.1.19.2.12.0.0.0.0.2.0.0.0.3.72.0", "readingTypeBrezObracuna": "0.0.4.1.19.2.12.0.0.0.0.2.0.0.0.3.72.0", "readingTypeObracun": "8.0.4.1.19.2.12.0.0.0.0.2.0.0.0.3.72.0", "tip": "delovna oddaja MT", "vrsta": "STANJE" }, { "naziv": "Prejeta jalova energija ET", "opis": "24 urno stanje, R+, kVArh, T0", "oznaka": "R+_T0", "perioda": "24 h", "readingType": "32.0.4.1.1.2.12.0.0.0.0.0.0.0.0.3.73.0", "readingTypeBrezObracuna": "0.0.4.1.1.2.12.0.0.0.0.0.0.0.0.3.73.0", "readingTypeObracun": "8.0.4.1.1.2.12.0.0.0.0.0.0.0.0.3.73.0", "tip": "jalova prejem ET", "vrsta": "STANJE" }, { "naziv": "Prejeta jalova energija VT", "opis": "24 urno stanje, R+, kVArh, T1", "oznaka": "R+_T1", "perioda": "24 h", "readingType": "32.0.4.1.1.2.12.0.0.0.0.1.0.0.0.3.73.0", "readingTypeBrezObracuna": "0.0.4.1.1.2.12.0.0.0.0.1.0.0.0.3.73.0", "readingTypeObracun": "8.0.4.1.1.2.12.0.0.0.0.1.0.0.0.3.73.0", "tip": "jalova prejem VT", "vrsta": "STANJE" }, { "naziv": "Prejeta jalova energija MT", "opis": "24 urno stanje, R+, kVArh, T2", "oznaka": "R+_T2", "perioda": "24 h", "readingType": "32.0.4.1.1.2.12.0.0.0.0.2.0.0.0.3.73.0", "readingTypeBrezObracuna": "0.0.4.1.1.2.12.0.0.0.0.2.0.0.0.3.73.0", "readingTypeObracun": "8.0.4.1.1.2.12.0.0.0.0.2.0.0.0.3.73.0", "tip": "jalova prejem MT", "vrsta": "STANJE" }, { "naziv": "Oddana jalova energija ET", "opis": "24 urno stanje, R-, kVArh, T0", "oznaka": "R-_T0", "perioda": "24 h", "readingType": "32.0.4.1.19.2.12.0.0.0.0.0.0.0.0.3.73.0", "readingTypeBrezObracuna": "0.0.4.1.19.2.12.0.0.0.0.0.0.0.0.3.73.0", "readingTypeObracun": "8.0.4.1.19.2.12.0.0.0.0.0.0.0.0.3.73.0", "tip": "jalova oddaja ET", "vrsta": "STANJE" }, { "naziv": "Oddana jalova energija VT", "opis": "24 urno stanje, R-, kVArh, T1", "oznaka": "R-_T1", "perioda": "24 h", "readingType": "32.0.4.1.19.2.12.0.0.0.0.1.0.0.0.3.73.0", "readingTypeBrezObracuna": "0.0.4.1.19.2.12.0.0.0.0.1.0.0.0.3.73.0", "readingTypeObracun": "8.0.4.1.19.2.12.0.0.0.0.1.0.0.0.3.73.0", "tip": "jalova oddaja VT", "vrsta": "STANJE" }, { "naziv": "Oddana jalova energija MT", "opis": "24 urno stanje, R-, kVArh, T2", "oznaka": "R-_T2", "perioda": "24 h", "readingType": "32.0.4.1.19.2.12.0.0.0.0.2.0.0.0.3.73.0", "readingTypeBrezObracuna": "0.0.4.1.19.2.12.0.0.0.0.2.0.0.0.3.73.0", "readingTypeObracun": "8.0.4.1.19.2.12.0.0.0.0.2.0.0.0.3.73.0", "tip": "jalova oddaja MT", "vrsta": "STANJE" } ]'
```

Comment:
messy. Pretty much added "enota": "xx" all over. Maybe it would be better to separate - was this initially hand written or is this something that is available somewhere from mojelektro?

Reply:
Yes, it's messy.
It's a json from mojelektro called `reading-type`, it's a fixed response, which i hardcoded to const to avoid additional calls for `reading-type`.... ALSO **there was a bug** with tag "oznaka" a while back in the original MojeElektro json.

For now i will not accept this pull, as `reading-type-array` should stay identical to MojeElektro response for future prof.